SwiftMessages/SwiftMessageModifier.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  SwiftMessageModifier.swift
+//  SwiftUIDemo
+//
+//  Created by Timothy Moose on 10/5/23.
+//
+
+import SwiftUI
+
+@available(iOS 14.0, *)
+public extension View {
+    /// A state-based modifier for displaying a message.
+    func swiftMessage<Message>(
+        message: Binding<Message?>,
+        config: SwiftMessages.Config? = nil,
+        swiftMessages: SwiftMessages? = nil
+    ) -> some View where Message: MessageViewConvertible {
+        modifier(SwiftMessageModifier(message: message))
```

Comment:
Looks like you forgot to pass the `config` and `swiftMessages` arguments through